PocketKit/Sources/PocketKit/Home/HomeCollectionViewSectionProvider.swift

```diff
@@ -124,3 +124,28 @@ class HomeViewControllerSectionProvider {
         return section
     }
 }
+
+
+extension HomeViewControllerSectionProvider {
+    func offlineSection(environment: NSCollectionLayoutEnvironment, withRecentSaves: Bool) -> NSCollectionLayoutSection {
+        var config = UICollectionLayoutListConfiguration(appearance: .plain)
+        config.backgroundColor = UIColor(.ui.white1)
+        config.showsSeparators = false
+        let section = NSCollectionLayoutSection.list(using: config, layoutEnvironment: environment)
+
+        let availableWidth = environment.container.contentSize.width
+        - ItemsListOfflineCell.Constants.padding
+        - ItemsListOfflineCell.Constants.padding
+
+        if !withRecentSaves {
+            section.contentInsets = NSDirectionalEdgeInsets(
+                top:  ItemsListOfflineCell.height(fitting: availableWidth) / 4, // Insets the top so "No Internet Connection" is centered
```

Comment:
I'm sure this is fine, but I'm just wondering why dividing by 4 happens to center this view?